### ISSUE TITLE
Remove obsolete branch?

### ISSUE THREAD

**Issue by neozhaoliang** [neozhaoliang/pywonderland#32]
@battleoftwok Hi, are you still maintaining this repo?

**Comment by indraine** [comment]
Yes, I did not particularly support this repository. Now I'm studying Rust, and I'm also writing programs in the aircraft industry.
